docs/Configuration.md

```diff
@@ -714,6 +714,77 @@ url: the URL of the GBFS feed (do not include the gbfs.json at the end) *
 
+#### TNC Configuration
+
+It is possible to add either Uber or Lyft as a TNC updater. OTP provides helper methods for making authenticated requests to each provider's API and also for verifying that TNC service exists when making certain routing requests that use a TNC for part or all of the trip.
+
+For each TNC updater, you have to manually determine what the ride type ids are for wheelchair-accessible services in the area that OTP will be planning trips in.
+
+##### Uber
+
+- Add one entry in the `updater` field of `router-config.json` in the format:
+
+```JSON
+{
+    "type": "transportation-network-company-updater",
+    "sourceType": "uber",
+    "serverToken": "YOUR-SECRET-TOKEN",
+    "wheelChairAccessibleRideType": "REGION-SPECIFIC-ID"
+}
+```
+
+##### Lyft
+
+- Add one entry in the `updater` field of `router-config.json` in the format:
+
+```JSON
+{
+    "type": "transportation-network-company-updater",
+    "sourceType": "lyft",
+    "clientId": "YOUR-CLIENT-ID-IF-LYFT-ISNT-BEING-ALL-WALLED-GARDEN-WITH-YOU",
+    "clientSecret": "YOUR-CLIENT-SECRET-IF-LYFT-ISNT-BEING-ALL-WALLED-GARDEN-WITH-YOU"
+}
+```
+
+#### Car Rental Configuration
+
+Currently, it is possible to add an updater for either car2go or the now defunct ReachNow. Both of these updaters read in data from a url or file about the vehicle positions and the areas where the cars are allowed to be dropped off inside of. The data format of the vehicle positions slightly differs between the each updater, but they're both a list of JSON objects. The regions data must be valid GeoJSON of Polygons or MultiPolygons in either a Feature or a Feature collection.
+
+Add one entry in the `updater` field of `router-config.json` in the format:
+
+```JSON
+{
+    "type": "car-rental-updater",
+    "sourceType": "car2go",
+    "frequencySec": 60,
+    "vehiclesUrl": "http://example.com/vehicles.json",
+    "regionsUrl": "http://example.com/regions.json"
+}
+```
+
+Use the same format above, but chagne the sourceType to `reachnow` and update the urls to setup for use with ReachNow.
+
+#### Micromobility Rental Configuration
+
+Adding networks of micromobility vehicles is very similar to adding in bikeshare, but the underlying code that determines how it is possible to pickup and dropoff vehicles and the speed at which they travel is different. Also, it is possible to add in data about where the vehicles can be dropped off at.
+
+The `network` field is used both internally and externally to identify the extent and use of this particular vehicle rental system. It's required for systems with more than 1 vehicle rental operator.
+
+The `url` field must point to the root `gbfs.json` endpoint.
+
+Add one entry in the `updater` field of `router-config.json` in the format:
+
+```JSON
+{
+    "type": "vehicle-rental-updater",
+    "frequencySec": 60,
+    "network": "RENTAL-SYSTEM-NAME",
```

Comment:
Might be nice to have an example value in a comment to the right of the `network` field.